gtsfm/evaluation/compare_metrics.py

```diff
@@ -0,0 +1,76 @@
+"""Function supporting metric comparisons across SfM pipelines (GTSfM, COLMAP, etc.)
+
+This function converts outputs from SfM pipelines into a format matching
+metrics from GTSfM.
+
+Authors: Jon Womack
+"""
+import os
+from typing import Dict, List
+
+import gtsfm.utils.io as io_utils
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
+
+import thirdparty.colmap.scripts.python.read_write_model as colmap_io
+
+
+def compare_metrics(
```

Comment:
also the function is not actually comparing, so we need a good name for it. something along "save_metrics_from_colmap_as_gtsfmmetricsgroup" but shorter.

Reply:
How about "save_other_pipelines_metrics" (the function isn't limited to colmap)?